src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  for (const key in robot) {
+    if (!robot[robot[key]]) {
+      robot[robot[key]] = key;
+      delete robot[key];
+    } else {
+      return null;
+    }
```

Comment:
You can invert your condition to avoid using `else` construction

Reply:
Fixed